pkg/scheduler/example_jobs.go

```diff
@@ -0,0 +1,41 @@
+package scheduler
+
+import (
+	"context"
+
+	"github.com/go-co-op/gocron/v2"
+	"go.uber.org/zap"
+
+	"github.com/cms-enterprise/easi-app/pkg/appcontext"
+)
+
+// exampleJobs is a struct that holds all the example jobs. It is not meant to be run in production
+// , but rather to show how to create a job.
+type exampleJobs struct {
+	RunEvery5SecondJob ScheduledJobWrapper[bool]
+	SimplifiedJob      ScheduledJob
+}
+
+var ExampleJobs = GetExampleJobs(SharedScheduler2)
+
+// GetExampleJobs returns a new exampleJobs struct with all the example jobs
+func GetExampleJobs(scheduler gocron.Scheduler) *exampleJobs {
+	return &exampleJobs{
+		SimplifiedJob:      NewScheduledJob("SimplifiedJob", scheduler, gocron.CronJob("*/5 * * * * *", true), simplifiedJobFunction),
+		RunEvery5SecondJob: NewScheduledJobWrapper("RunEverySecondJob", scheduler, gocron.CronJob("*/5 * * * * *", true), runEvery5SecondJobFunction, true),
+	}
+}
```

Comment:
```suggestion
func GetExampleJobs(scheduler gocron.Scheduler) *exampleJobs {
    cronExpression := "*/5 * * * * *"
	return &exampleJobs{
		SimplifiedJob:      NewScheduledJob("SimplifiedJob", scheduler, gocron.CronJob(cronExpression, true), simplifiedJobFunction),
		RunEvery5SecondJob: NewScheduledJobWrapper("RunEverySecondJob", scheduler, gocron.CronJob(cronExpression, true), runEvery5SecondJobFunction, true),
	}
}
```

Reply:
good call, thanks!

Reply:
refactored to use the timing package! So any standard timing can call the job definitions from that package instead :D